### ISSUE TITLE
git-lfs.pkg recipe installs things to "wrong" locations.

### ISSUE THREAD

**Issue by gregneagle** [autopkg/apettinen-recipes#44]
Currently the git-lfs.pkg recipe seems to just put the entire unzipped archive into /usr/local/bin:
```
$ pkgutil --payload-files git-lfs-amd64-3.2.0.pkg 
.
./usr
./usr/local
./usr/local/bin
./usr/local/bin/git-lfs
./usr/local/bin/git-lfs/git-lfs
./usr/local/bin/git-lfs/man
<snip>
./usr/local/bin/git-lfs/README.md
```

I'd expect the git-lfs binary at
```
./usr/local/bin/git-lfs
```
and the then the contents of the man/man1 and man/man5 directories installed under
```
./usr/local/man/man1
./usr/local/man/man5
```
etc.
As it is currently, git-lfs doesn't actually work post-install because it's installed to `/usr/local/bin/git-lfs/git-lfs`. And the man pages don't work, either.

**Comment by gregneagle** [comment]
I'm working on a PR to address this issue.

**Comment by apettinen** [comment]
Thanks Greg for this, appreciated.

Fixed by b0ab91eaaab656fb935a3993f3aabd171472d6c7
